packages/connect/package.json

```diff
@@ -24,7 +24,10 @@
     "test:inspect": "node --inspect ../../node_modules/.bin/jest --watch --no-cache --runInBand"
   },
   "dependencies": {
-    "@frontity/error": "^0.1.3"
+    "@frontity/react-easy-state": "7.0.0",
+    "@frontity/observer-util": "4.3.0",
```

Comment:
Do you feel strongly about having the versions locked? Since these are gonna be maintained by us, we might be able to use the `^` flag for the versions. Just as we do with the current packages.

Reply:
Ups, good catch. Yeah, you are right. I had them fixed because they were pointing to a beta before, but I forgot to unfix them 👍

Reply:
Done!